workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/InviteMemberModal/InviteMemberModal.tsx

```diff
@@ -161,14 +162,19 @@ export function InviteMemberModal({
     await addExistentUserIntoAccount(selectedAccount.id, formFields.email);
     await addAccountRolesToUser(user);
 
+    const accountGroups = await getAccountGroup(selectedAccount.id);
+
+    const accountGroupERC = accountGroups && accountGroups[0]?.externalReferenceCode;
```

Comment:
Did you know that in case of accountGroups doesn't exist, the value of this variable will be false? is that the expected behavior? 

Consider to use

```suggestion
    const accountGroupERC = accountGroups[0]?.externalReferenceCode ?? "";
```